src/sentry/static/sentry/app/views/performance/vitalDetail/vitalChart.tsx

```diff
@@ -0,0 +1,302 @@
+import {Client} from 'app/api';
+import {t} from 'app/locale';
+import LineChart from 'app/components/charts/lineChart';
+import ChartZoom from 'app/components/charts/chartZoom';
+import ErrorPanel from 'app/components/charts/errorPanel';
+import TransparentLoadingMask from 'app/components/charts/transparentLoadingMask';
+import TransitionChart from 'app/components/charts/transitionChart';
+import EventsRequest from 'app/components/charts/eventsRequest';
+import ReleaseSeries from 'app/components/charts/releaseSeries';
+import QuestionTooltip from 'app/components/questionTooltip';
+import {getInterval, getSeriesSelection} from 'app/components/charts/utils';
+import {IconWarning} from 'app/icons';
+import {getUtcToLocalDateObject} from 'app/utils/dates';
+import EventView from 'app/utils/discover/eventView';
+import withApi from 'app/utils/withApi';
+import {decodeScalar} from 'app/utils/queryString';
+import theme from 'app/utils/theme';
+import {tooltipFormatter, axisLabelFormatter} from 'app/utils/discover/charts';
+import getDynamicText from 'app/utils/getDynamicText';
+import {Panel} from 'app/components/panels';
+import styled from 'app/styled';
+import space from 'app/styles/space';
+import MarkLine from 'app/components/charts/components/markLine';
+
+import {getMaxOfSeries, vitalChartTitleMap, vitalNameFromLocation} from './utils';
+import {HeaderTitleLegend} from '../styles';
+import {WEB_VITAL_DETAILS} from '../transactionVitals/constants';
+import {
+  replaceSeriesName,
+  replaceSmoothedSeriesName,
+  transformEventStatsSmoothed,
+} from '../trends/utils';
+
+const QUERY_KEYS = [
+  'environment',
+  'project',
+  'query',
+  'start',
+  'end',
+  'statsPeriod',
+] as const;
+
+type ViewProps = Pick<EventView, typeof QUERY_KEYS[number]>;
+
+type Props = ReactRouter.WithRouterProps &
+  ViewProps & {
+    api: Client;
+    location: Location;
+    organization: OrganizationSummary;
+  };
+
+class VitalChart extends React.Component<Props> {
+  handleLegendSelectChanged = legendChange => {
+    const {location} = this.props;
+    const {selected} = legendChange;
+    const unselected = Object.keys(selected).filter(key => !selected[key]);
+
+    const to = {
+      ...location,
+      query: {
+        ...location.query,
+        unselectedSeries: unselected,
+      },
+    };
+    browserHistory.push(to);
+  };
+
+  render() {
+    const {
+      api,
+      project,
+      environment,
+      location,
+      organization,
+      query,
+      statsPeriod,
+      router,
+    } = this.props;
+
+    const start = this.props.start
+      ? getUtcToLocalDateObject(this.props.start)
+      : undefined;
+
+    const end = this.props.end ? getUtcToLocalDateObject(this.props.end) : undefined;
+    const utc = decodeScalar(router.location.query.utc);
+
+    const vitalName = vitalNameFromLocation(location);
+    const chartTitle = vitalChartTitleMap[vitalName];
+
+    const yAxis = [`p50(${vitalName})`, `p75(${vitalName})`];
```

Comment:
Would it be good to match transaction summary here and plot more than just p50 and p75?

Reply:
nah, I think we mostly want to show the passing here, this is somewhat a placeholder until we get %, but it basically shows p75 vs. threshold. p50 is just here for flavor.